src/Account.sol

```diff
@@ -743,50 +757,46 @@ contract Account is IAccount, OpsReady, Owned, Initializable {
     /// @notice fetch PerpsV2Market market defined by market key
     /// @param _marketKey: key for Synthetix PerpsV2 market
     /// @return IPerpsV2Market contract interface
-    function getPerpsV2Market(bytes32 _marketKey)
+    function _getPerpsV2Market(bytes32 _marketKey)
         internal
         view
         returns (IPerpsV2MarketConsolidated)
     {
-        return
-            IPerpsV2MarketConsolidated(
-                futuresMarketManager.marketForKey(_marketKey)
-            );
+        return IPerpsV2MarketConsolidated(futuresMarketManager.marketForKey(_marketKey));
     }
 
     /// @notice get exchange rate of underlying market asset in terms of sUSD
     /// @param _market: Synthetix PerpsV2 Market
     /// @return price in sUSD
-    function sUSDRate(IPerpsV2MarketConsolidated _market)
-        internal
-        view
-        returns (uint256)
-    {
+    function _sUSDRate(IPerpsV2MarketConsolidated _market) internal view returns (uint256) {
         (uint256 price, bool invalid) = _market.assetPrice();
         if (invalid) {
             revert InvalidPrice();
         }
         return price;
     }
 
-    /// @notice exchangeRates() fetches current ExchangeRates contract
-    function exchanger() internal view returns (IExchanger) {
-        return
-            IExchanger(
-                ADDRESS_RESOLVER.requireAndGetAddress(
-                    "Exchanger",
-                    "Account: Could not get Exchanger"
-                )
-            );
-    }
-
     /*//////////////////////////////////////////////////////////////
                              MATH UTILITIES
     //////////////////////////////////////////////////////////////*/
 
-    /// @notice Absolute value of the input, returned as an unsigned number.
+    /// @notice get absolute value of the input, returned as an unsigned number.
+    /// @param x: signed number
+    /// @return z uint256 absolute value of x
+    function _abs(int256 x) internal pure returns (uint256 z) {
+        assembly {
+            let mask := sub(0, shr(255, x))
+            z := xor(mask, add(mask, x))
+        }
+    }
+
+    /// @notice determines if input numbers have the same sign
+    /// @dev asserts that both numbers are not zero
     /// @param x: signed number
-    function _abs(int256 x) internal pure returns (uint256) {
-        return uint256(x < 0 ? -x : x);
+    /// @param y: signed number
+    /// @return true if same sign, false otherwise
+    function _isSameSign(int256 x, int256 y) internal pure returns (bool) {
+        assert(x != 0 && y != 0);
```

Comment:
Technically, 0 == 0 is the same sign no?

Reply:
0 has no sign, according to google. So if no sign then they cannot be the same